run/anubis@.service

```
@@ -5,8 +5,14 @@ Description="Anubis HTTP defense proxy (instance %i)"
 ExecStart=/usr/bin/anubis
 Restart=always
 RestartSec=30s
-EnvironmentFile=/etc/anubis/anubis-%i.env
+EnvironmentFile=/etc/anubis/%i.env
 LimitNOFILE=infinity
+DynamicUser=1
```

```suggestion
DynamicUser=yes
```
I think that somehow gets mangled into a boolean but usually that is yes/true or no/false